diff --git a/scripts/properties.sh b/scripts/properties.sh
old mode 100644
new mode 100755
index b98c3a4690..fa14d614d6
--- a/scripts/properties.sh
+++ b/scripts/properties.sh
@@ -76,7 +76,7 @@ __TERMUX_BUILD_PROPS__VALIDATE_PATHS_MAX_LEN="true"
 # Whether to validate `usr` merge format for `TERMUX__PREFIX`. Set to `false` to skip validation.
 # Check `TERMUX__PREFIX` variable docs for more info.
 ##
-__TERMUX_BUILD_PROPS__VALIDATE_TERMUX_PREFIX_USR_MERGE_FORMAT="true"
+__TERMUX_BUILD_PROPS__VALIDATE_TERMUX_PREFIX_USR_MERGE_FORMAT="false"
 
 
 
@@ -371,7 +371,7 @@ fi
 #
 # Default value: `Termux`
 ##
-TERMUX__NAME="Termux"
+TERMUX__NAME="AutoPie"
 
 ##
 # The lower case value for `TERMUX__NAME`.
@@ -464,7 +464,7 @@ TERMUX__REPOS_HOST_ORG_URL="https://github.com/$TERMUX__REPOS_HOST_ORG_NAME"
 #
 # Default value: `com.termux`
 ##
-TERMUX_APP__PACKAGE_NAME="com.termux"
+TERMUX_APP__PACKAGE_NAME="com.autosec.pie"
 TERMUX_APP_PACKAGE="$TERMUX_APP__PACKAGE_NAME" # Deprecated alternative variable for `TERMUX_APP__PACKAGE_NAME`
 
 __termux_build_props__add_variables_validator_actions "TERMUX_APP__PACKAGE_NAME" "app_package_name"
@@ -743,7 +743,7 @@ __termux_build_props__add_variables_validator_actions "TERMUX__ROOTFS_ID" "unsig
 #
 # Default value: `files`
 ##
-TERMUX__ROOTFS_SUBDIR="files"
+TERMUX__ROOTFS_SUBDIR="files/build"
 __termux_build_props__add_variables_validator_actions "TERMUX__ROOTFS_SUBDIR" "allow_unset_value safe_relative_path"
 
 ###########
